src/index.slim

```diff
@@ -97,7 +93,7 @@ html
         }
 
         updateUnit(result) {
-          if (result < this.carbonTarget) {
+          if (result < 2) {
```

Comment:
pourquoi c'est 2 en dur et pas la valeur de `this.goalTarget` ou qqchose comme ça ?

Reply:
Ben parce qu'il s'agit de décider d'un pluriel... et qu'un pluriel se décide si c'est en dessous ou au dessus de 2, pas sur une valeur qui peut changer.

C'est un Hazard si ça fonctionnait bien avant, mais le code précédent n'avait pas de sens.

Reply:
ah oui lol, c'est juste pour un pluriel, BIEN VU